substrate/frame/nis/src/tests.rs

```diff
@@ -756,15 +758,15 @@ fn enlargement_to_target_works() {
 			ReceiptRecord {
 				proportion: Perquintill::from_percent(10),
 				owner: Some((2, 40)),
-				expiry: 13
+				expiry: 12
```

Comment:
Why did this change?

Reply:
@bkchr `on_initialize` logic now tracks processing relative to the `LastProcessedBlock` (not modulo IntakePeriod).

Reply:
@bkchr please review